### ISSUE TITLE
Server Crash with JAVA 8

### ISSUE THREAD

**Issue by Landstryder** [AnimeniacYuuto/EnhancedInventories#29]
---- Minecraft Crash Report ----
// This doesn't make any sense!

Time: 03/08/15 04:12
Description: Exception in server tick loop

cpw.mods.fml.common.LoaderException: java.lang.ClassNotFoundException: yuuto.enhancedinventories.EnhancedInventories
	at cpw.mods.fml.common.LoadController.transition(LoadController.java:163)
	at cpw.mods.fml.common.Loader.loadMods(Loader.java:544)
	at cpw.mods.fml.server.FMLServerHandler.beginServerLoading(FMLServerHandler.java:87)
	at cpw.mods.fml.common.FMLCommonHandler.onServerStart(FMLCommonHandler.java:314)
	at net.minecraft.server.dedicated.DedicatedServer.func_71197_b(DedicatedServer.java:117)
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:387)
	at net.minecraft.server.MinecraftServer$2.run(MinecraftServer.java:685)
Caused by: java.lang.ClassNotFoundException: yuuto.enhancedinventories.EnhancedInventories
	at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:191)
	at java.lang.ClassLoader.loadClass(Unknown Source)
	at java.lang.ClassLoader.loadClass(Unknown Source)
	at cpw.mods.fml.common.ModClassLoader.loadClass(ModClassLoader.java:58)
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Unknown Source)
	at cpw.mods.fml.common.FMLModContainer.constructMod(FMLModContainer.java:440)
	at sun.reflect.GeneratedMethodAccessor4.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74)
	at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47)
	at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322)
	at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304)
	at com.google.common.eventbus.EventBus.post(EventBus.java:275)
	at cpw.mods.fml.common.LoadController.sendEventToModContainer(LoadController.java:212)
	at cpw.mods.fml.common.LoadController.propogateStateMessage(LoadController.java:190)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74)
	at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47)
	at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322)
	at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304)
	at com.google.common.eventbus.EventBus.post(EventBus.java:275)
	at cpw.mods.fml.common.LoadController.distributeStateMessage(LoadController.java:119)
	at cpw.mods.fml.common.Loader.loadMods(Loader.java:513)
	... 5 more
Caused by: java.lang.NullPointerException
	at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:182)
	... 32 more

A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- System Details --
Details:
	Minecraft Version: 1.7.10
	Operating System: Windows Server 2008 R2 (amd64) version 6.1
	Java Version: 1.8.0_45, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 643645528 bytes (613 MB) / 1425014784 bytes (1359 MB) up to 3728736256 bytes (3556 MB)
	JVM Flags: 2 total; -XX:MaxPermSize=256m -Xmx4000M
	AABB Pool Size: 0 (0 bytes; 0 MB) allocated, 0 (0 bytes; 0 MB) used
	IntCache: cache: 0, tcache: 0, allocated: 0, tallocated: 0
	FML: MCP v9.05 FML v7.10.99.99 Minecraft Forge 10.13.4.1448 130 mods loaded, 130 mods active
	States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored
	UC	mcp{9.05} [Minecraft Coder Pack] (minecraft.jar) 
	UC	FML{7.10.99.99} [Forge Mod Loader] (FTBServer.jar) 
	UC	Forge{10.13.4.1448} [Minecraft Forge] (FTBServer.jar) 
	UC	AppleMilkCore{1.1.0} [AppleMilkCore] (minecraft.jar) 
	UC	CodeChickenCore{1.0.6.43} [CodeChicken Core] (minecraft.jar) 
	UC	Micdoodlecore{} [Micdoodle8 Core] (minecraft.jar) 
	UC	NotEnoughItems{1.0.4.107} [Not Enough Items] (NotEnoughItems-1.7.10-1.0.4.107-universal.jar) 
	UC	OpenComputers|Core{1.5.14.30} [OpenComputers (Core)] (minecraft.jar) 
	UC	PlayerAPI{1.3} [Player API] (minecraft.jar) 
	UC	PotionExtensionCore{1.7.10-1.1.0} [PotionExtensionCore] (minecraft.jar) 
	UC	OpenModsCore{0.7.3} [OpenModsCore] (minecraft.jar) 
	UC	<CoFH ASM>{000} [CoFH ASM] (minecraft.jar) 
	UC	debug{1.0} [debug] (denseores-1.6.2.jar) 
	UC	DCsAppleMilk{1.7.10_2.8b} [Apple&Milk&Tea!] (AppleMilkTea2-1.7.10_2.8b.jar) 
	UC	AWWayofTime{v1.3.3} [Blood Magic: Alchemical Wizardry] (BloodMagic-1.7.10-1.3.3-10.jar) 
	UC	amtgen{1.4.2} [AMT Generators] (AMT_Generators-1.7.10-1.4.2.jar) 
	UC	EE3{0.3.507} [Equivalent Exchange 3] (EquivalentExchange3-1.7.10-0.3.507.jar) 
	UC	ForgeMultipart{1.2.0.344} [Forge Multipart] (ForgeMultipart-1.7.10-1.2.0.344-universal.jar) 
	UC	chisel{2.4.1.40} [Chisel 2] (Chisel2-2.4.1.40.jar) 
	UC	koresample{1.7.10-1.3.2} [Kore Sample] (KoreSample-1.7.10-1.3.2.jar) 
	UC	StorageDrawers{1.7.10-1.5.12} [Storage Drawers] (StorageDrawers-1.7.10-1.5.12.jar) 
	UC	dendrology{1.7.10-1.6.2} [Ancient Trees] (AncientTrees-1.7.10-1.6.2.jar) 
	UC	aobdbb{1.0.4} [Another One Bites The Dust: Berry Bushes] (AOBDBB-1.0.4.jar) 
	UC	AppleCore{1.1.0} [AppleCore] (AppleCore-mc1.7.10-1.1.0.jar) 
	UC	DCsNEIPluginAMT{1.7.10_1.0a} [NEIPluginForAMT] (AppleMilkTea2-1.7.10_2.8b.jar) 
	UC	CoFHCore{1.7.10R3.0.3B2} [CoFH Core] (CoFHCore-[1.7.10]3.0.3B2-294.jar) 
	UC	asielib{0.4.0} [asielib] (AsieLib-1.7.10-0.4.0.jar) 
	UC	Baubles{1.0.1.10} [Baubles] (Baubles-1.7.10-1.0.1.10.jar) 
	UC	aura{0.1r0} [Aura Cascade] (AuraCascade-526.jar) 
	UC	autopackager{1.5.2} [AutoPackager] (autopackager-1.5.3.jar) 
	UC	bdlib{1.9.0.94} [BD Lib] (bdlib-1.9.0.94-mc1.7.10.jar) 
	UC	Waila{1.5.10} [Waila] (Waila-1.5.10_1.7.10.jar) 
	UC	BloodArsenal{1.1-14} [Blood Arsenal] (BloodArsenal-1.7.10-1.1-14.jar) 
	UC	qmunitylib{1.0} [QmunityLib] (QmunityLib-1.7.10-0.1.114-universal.jar) 
	UC	bluepower{0.2.954} [Blue Power] (BluePower-1.7.10-0.2.954-universal.jar) 
	UC	CarpentersBlocks{3.3.6} [Carpenter's Blocks] (Carpenter's Blocks v3.3.6 - MC 1.7.10.jar) 
	UC	ChickenChunks{1.3.4.19} [ChickenChunks] (ChickenChunks-1.7.10-1.3.4.19-universal.jar) 
	UC	ctmlib{1.0.1.5} [CTMLib] (Chisel2-2.4.1.40.jar) 
	UC	Ztones{1.7.10} [Ztones] (Ztones-1.7.10-2.2.1.jar) 
	UC	chiseltones{@VERSION@} [Chisel Tones] (ChiselTones-1.7.10-1.0-1.jar) 
	UC	clockworkphase{1.7.10_1.0f} [Clockwork Phase] (clockworkphase-1.7.10_1.0f.jar) 
	UC	EnderStorage{1.4.7.36} [EnderStorage] (EnderStorage-1.7.10-1.4.7.36-universal.jar) 
	UC	OpenComputers{1.5.14.30} [OpenComputers] (OpenComputers-MC1.7.10-1.5.14.30-universal.jar) 
	UC	GalacticraftCore{3.0.12} [Galacticraft Core] (GalacticraftCore-1.7-3.0.12.350.jar) 
	UC	Mekanism{8.1.5} [Mekanism] (Mekanism-1.7.10-8.1.5.246.jar) 
	UC	computronics{1.5.0} [Computronics] (Computronics-1.7.10-1.5.0.jar) 
	UC	props{2.0.2} [Decocraft] (Decocraft-2.0.2_1.7.10.jar) 
	UC	DummyCore{1.13} [DummyCore] (DummyCore1.13.jar) 
	UC	EE3HELP{2.2a} [EE3 Helper] (EE3Help-2.2a.jar) 
	UC	Emotes{1.0} [Emotes] (Emotes 1.0.1.jar) 
	UC	eng_toolbox{1.2.3.0} [Engineer's Toolbox] (EngineersToolbox-1.2.3.0-BETA.jar) 
	UC	RefinedRelocation{1.0.8b} [Refined Relocation] (RefinedRelocation-1.7.10-1.0.8b.jar) 
	UE	EnhancedInventories{1.7.10-1.0.13} [Enhanced Inventories] (EnhancedInventories-1.7.10-1.0.13.jar) 
	UC	essentialcraft{4.6.1710.35} [EssentialCraftIII] (EssentialCraftv4.6.1710.35.jar) 
	UC	ThermalFoundation{1.7.10R1.2.0B1} [Thermal Foundation] (ThermalFoundation-[1.7.10]1.2.0B1-95.jar) 
	UC	ExtraUtilities{1.2.8} [Extra Utilities] (extrautilities-1.2.8.jar) 
	UC	farseek{1.0.8} [farseek] (Farseek-1.0.8.jar) 
	UC	FastCraft{1.21} [FastCraft] (fastcraft-1.21.jar) 
	UC	Steamcraft{0.28.7} [Flaxbeard's Steam Power] (Flaxbeard'sSteamPower-1.7.10-0.28.7.jar) 
	UC	FloodLights{0.4.3} [Flood Lights] (FloodLights-1.7.10-0.4.3-62.jar) 
	UC	framez{0.2} [Framez] (Framez-1.7.10-0.2-32-universal.jar) 
	UC	GalacticraftMars{3.0.12} [Galacticraft Planets] (Galacticraft-Planets-1.7-3.0.12.350.jar) 
	UC	GardenCore{1.7.10-1.6.3} [Garden Core] (GardenStuff-1.7.10-1.6.3.jar) 
	UC	GardenContainers{1.7.10-1.6.3} [Garden Containers] (GardenStuff-1.7.10-1.6.3.jar) 
	UC	GardenStuff{1.7.10-1.6.3} [Garden Stuff] (GardenStuff-1.7.10-1.6.3.jar) 
	UC	GardenTrees{1.7.10-1.6.3} [Garden Trees] (GardenStuff-1.7.10-1.6.3.jar) 
	UC	GekosLasers{1.7.10R0.8} [Geko's Lasers] (GekosLasers-1.7.10-0.8.jar) 
	UC	advgenerators{0.9.14.89} [Advanced Generators] (generators-0.9.14.89-mc1.7.10.jar) 
	UC	guideapi{1.7.10-1.0.1-20} [Guide-API] (Guide-API-1.7.10-1.0.1-20.jar) 
	UC	HardcoreEnderExpansion{1.8.2} [Hardcore Ender Expansion] (HardcoreEnderExpansion  MC-1.7.10  v1.8.2.jar) 
	UC	headcrumbs{1.3.1} [Headcrumbs] (Headcrumbs-1.3.1.jar) 
	UC	ibench{1.0.12} [iBench] (ibench-1.7.10-1.0.12.jar) 
	UC	IGWMod{1.1.3-18} [In-Game wiki Mod] (IGW-Mod-1.7.10-1.1.3-18-universal.jar) 
	UC	ImmersiveEngineering{0.4.4} [Immersive Engineering] (ImmersiveEngineering-0.4.4.jar) 
	UC	immersiveintegration{0.4.1} [Immersive Integration] (immersiveintegration-0.4.1.jar) 
	UC	inventorytweaks{1.59-dev-152-cf6e263} [Inventory Tweaks] (InventoryTweaks-1.59-dev-152.jar) 
	UC	ironbackpacks{0.5.0} [Iron Backpacks] (Iron Backpacks-1.7.10-1.0.4-universal.jar) 
	UC	LatCoreMC{1.6.9} [LatCoreMC] (LatCoreMC-1.7.10-1.6.9.4.jar) 
	UC	LatBlocks{1.6.9} [LatBlocks] (LatBlocks-1.7.10-1.6.9.2.jar) 
	UC	LomLib{3.2.0} [LomLib] (LomLib-1.7.10-3.2.0-universal.jar) 
	UC	magnanimoustools{0.1.6} [Magnanimous Tools] (MagTools-1.7.10-0.1.6.jar) 
	UC	malisiscore{1.7.10-0.11.4} [MalisisCore] (malisiscore-1.7.10-0.11.4.jar) 
	UC	malisisdoors{1.7.10-1.6} [Malisis' Doors] (malisisdoors-1.7.10-1.6.jar) 
	UC	Mantle{1.7.10-0.3.2.jenkins184} [Mantle] (Mantle-1.7.10-0.3.2.jar) 
	UC	mo{0.4.0-RC2} [Matter Overdrive] (MatterOverdrive-1.7.10-0.4.0-RC2.jar) 
	UC	MekanismGenerators{8.1.5} [MekanismGenerators] (MekanismGenerators-1.7.10-8.1.5.246.jar) 
	UC	MekanismTools{8.1.5} [MekanismTools] (MekanismTools-1.7.10-8.1.5.246.jar) 
	UC	MineTweaker3{3.0.9} [MineTweaker 3] (MineTweaker3-1.7.10-3.0.9.jar) 
	UC	testdummy{1.0} [MmmMmmMmmMmm] (MmmMmmMmmMmm-1.6.jar) 
	UC	ModularArmour{1.0.0} [Modular Armour] (ModularArmour-1.7.10-1.0.15-38.jar) 
	UC	Morpheus{1.7.10-1.6.8} [Morpheus] (Morpheus-1.7.10-1.6.8.jar) 
	UC	NEIAddons{1.12.8.30} [NEI Addons] (neiaddons-1.12.8.30-mc1.7.10.jar) 
	UC	NEIAddons|AppEng{1.12.8.30} [NEI Addons: Applied Energistics 2] (neiaddons-1.12.8.30-mc1.7.10.jar) 
	UC	NEIAddons|Botany{1.12.8.30} [NEI Addons: Botany] (neiaddons-1.12.8.30-mc1.7.10.jar) 
	UC	NEIAddons|Forestry{1.12.8.30} [NEI Addons: Forestry] (neiaddons-1.12.8.30-mc1.7.10.jar) 
	UC	NEIAddons|CraftingTables{1.12.8.30} [NEI Addons: Crafting Tables] (neiaddons-1.12.8.30-mc1.7.10.jar) 
	UC	NEIAddons|ExNihilo{1.12.8.30} [NEI Addons: Ex Nihilo] (neiaddons-1.12.8.30-mc1.7.10.jar) 
	UC	neiintegration{1.0.9} [NEI Integration] (NEIIntegration-MC1.7.10-1.0.9.jar) 
	UC	neresources{0.1.0.95} [Not Enough Resources] (NotEnoughResources-1.7.10-0.1.0.95.jar) 
	UC	ObsidiPlates{3.0.0.18} [ObsidiPlates] (ObsidiPlates-1.7.10-universal-3.0.0.18.jar) 
	UC	OpenMods{0.7.3} [OpenMods] (OpenModsLib-1.7.10-0.7.3.jar) 
	UC	OpenBlocks{1.4.3} [OpenBlocks] (OpenBlocks-1.7.10-1.4.3.jar) 
	UC	openmodularturrets{2.1.0-151} [Open Modular Turrets] (OpenModularTurrets-1.7.10-2.1.0-151.jar) 
	UC	OpenPeripheralCore{1.1.1} [OpenPeripheralCore] (OpenPeripheralCore-1.7.10-1.1.1.jar) 
	UC	OpenPeripheral{0.3.1} [OpenPeripheralAddons] (OpenPeripheralAddons-1.7.10-0.3.1.jar) 
	UC	OpenPeripheralIntegration{0.2.2} [OpenPeripheralIntegration] (OpenPeripheralIntegration-1.7.10-0.2.2.jar) 
	UC	particlephysics{@MAJOR@.@MINOR@.@BUILD@} [Particle Physics] (ParticlePhysics-1.7.10-1.0.44.jar) 
	UC	progressiveautomation{1.6.21} [Progressive Automation] (ProgressiveAutomation-1.7.10-1.6.21.jar) 
	UC	quantumflux{1.7.10-1.2.3} [QuantumFlux] (QuantumFlux-1.7.10-1.2.3.jar) 
	UC	quiverchevsky{b100} [QuiverBow] (QuiverBow_1.7.10_b100.zip) 
	UC	RandomThings{2.2.4} [Random Things] (RandomThings-2.2.4.jar) 
	UC	Redstonic{1.4.8} [Redstonic] (Redstonic-v1.4.8.jar) 
	UC	rftools{3.11} [RFTools] (rftools-3.11.jar) 
	UC	rfwindmill{0.7a} [RF Windmill] (rfwindmill-1.7.10-0.7a.jar) 
	UC	RouterReborn{1.2.0.31} [Router Reborn] (RouterReborn-1.7.10-1.2.0.31-universal.jar) 
	UC	RWG{Alpha 1.3.2} [Realistic World Gen Alpha] (RWG-alpha-1.3.2.jar) 
	UC	Sanguimancy{1.7.10-1.1.9-31} [Sanguimancy] (Sanguimancy-1.7.10-1.1.9-31.jar) 
	UC	SanguineExtras{1.3.2} [Sanguine Utilities] (Sanguine Utilities-1.3.2.jar) 
	UC	Silicio{1.6.9} [Silicio] (Silicio-1.7.10-1.6.9.2.jar) 
	UC	simplecondenser{1.2.0} [Simple Condensers] (Simple Condensers-1.7.10-1.2.0-universal.jar) 
	UC	SSTOW{1.7.10-0.1-RC9-7} [Soul Shards: The Old Ways] (SoulShards-TOW-1.7.10-0.1-RC9-7.jar) 
	UC	streams{0.1.4} [streams] (Streams-0.1.4.jar) 
	UC	ttCore{MC1.7.10-0.1.1-77} [ttCore] (ttCore-MC1.7.10-0.1.1-77.jar) 
	UC	wawla{1.2.1} [What Are We Looking At] (Wawla-1.2.1_1.7.10.jar) 
	UC	wiiemc{1.0.1} [What Is Its EMC?] (WIIEMC-1.7.10-1.0.1-universal.jar) 
	UC	XPT{1.0.4} [XP Teleporters] (XPTeleporters-1.7.10-1.0.4.jar) 
	UC	McMultipart{1.2.0.344} [Minecraft Multipart Plugin] (ForgeMultipart-1.7.10-1.2.0.344-universal.jar) 
	UC	aobd{2.7.1} [Another One Bites The Dust] (AOBD-2.7.1.jar) 
	UC	denseores{1.0} [Dense Ores] (denseores-1.6.2.jar) 
	UC	ForgeMicroblock{1.2.0.344} [Forge Microblocks] (ForgeMultipart-1.7.10-1.2.0.344-universal.jar) 
	OpenModsLib class transformers: [stencil_patches:ENABLED],[movement_callback:ENABLED],[map_gen_fix:ENABLED],[gl_capabilities_hook:ENABLED],[player_render_hook:ENABLED]
	Class transformer null safety: all safe
	CoFHCore: -[1.7.10]3.0.3B2-294
	ThermalFoundation: -[1.7.10]1.2.0B1-95
	Mantle Environment: Environment healthy.
	Profiler Position: N/A (disabled)
	Is Modded: Definitely; Server brand changed to 'fml,forge'
	Type: Dedicated Server (map_server.txt)

**Comment by AnimeniacYuuto** [comment]
can i get a forge log please? This is a weird error

**Comment by Landstryder** [comment]
Well this is crazy... the other night when I had this issue I had updated the server to JAVA 8 and loaded the new 1.1.0 version of Horizons... I also loaded the pack on my curse launcher and manually added the Enhanced Inventories mod through the curse launcher. I then used that file to added it to the server mod directory. After the first time it crashed I even tried to re-upload the file again but had the same result.... Today after reading your post I decided to download the file fresh from curse using my browser and uploaded that file as you suggested, and guess what the error cleared... The whole thing was odd because my client worked fine with the file it loaded through the launcher but it must have got corrupted both times I tried uploading it from there to the server... Anyway keep up the good work, and thanks for watching my series. Sorry for the false alarm.

p.s. I think it would be awesome if you made a block like the door factory from Malisis Door, that would do the same thing with your inventories.. select the trim, top and side textures from any block.
